.buildkite/release-package-registry-distribution.yml

```
@@ -1,6 +1,37 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
+env:
+  SETUP_GOLANG_VERSION: '1.20.4'
+  DOCKER_REGISTRY: 'docker.elastic.co'
+
 steps:
-  - label: ":golangci-lint: Checks"
+  - input: "Input version for docker tag"
+    fields:
+      - text: "The docker tag to be published (format: major.minor.patch(-prerelease)?)."
+        hint: "e.g. 0.0.1"
+        key: "DOCKER_TAG_VERSION"
+        required: true
+        default: "latest"
+  - wait
+
+  - label: ":hammer: Validate docker tag"
+    key: validate
+    command:
+      - ".buildkite/scripts/validate-tag.sh"
+
+  - label: ":hammer: Release registry distribution lite"
```

Could this label show the `tag_name`? 
Would this work ? 

```suggestion
  - label: ":hammer: Release registry distribution {{max.tag_name}}"
```